lib/auth/auth_test.go

```diff
@@ -0,0 +1,33 @@
+package auth
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestNewToken(t *testing.T) {
```

Comment:
It will be better from readability and maintainability PoV splitting this function into two separate functions - TestNewTokenSuccess and TestNewTokenFailure

Reply:
updated